docker/docker-compose.yml

```diff
@@ -60,7 +60,6 @@ services:
       ENABLE_TELEMETRY: ${ENABLE_TELEMETRY:-true}
       RW_TELEMETRY_TYPE: ${RW_TELEMETRY_TYPE:-"docker-compose"}
       RW_SECRET_STORE_PRIVATE_KEY_HEX: ${RW_SECRET_STORE_PRIVATE_KEY_HEX:-0123456789abcdef}
-      RW_LICENSE_KEY: ${RW_LICENSE_KEY:-""}
```

Comment:
I think we should still pass the env var? ...but only remove the quotes.

```yaml
RW_LICENSE_KEY: ${RW_LICENSE_KEY:-}
```

Reply:
cannot tell the diff here. But after removing the line, it works.

Reply:
Only env vars specified here will inherit from the host.